universal-application-tool-0.0.1/app/services/aws/AwsRegion.java

```diff
@@ -0,0 +1,27 @@
+package services.aws;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.typesafe.config.Config;
+import javax.inject.Inject;
+import software.amazon.awssdk.regions.Region;
+
+public class AwsRegion {
+  public static final String AWS_REGION_CONF_PATH = "aws.region";
+
+  private final Region region;
+
+  @Inject
+  public AwsRegion(Config config) {
+    String regionName = checkNotNull(config).getString(AWS_REGION_CONF_PATH);
+    region = Region.of(regionName);
+  }
+
+  public Region get() {
+    return region;
+  }
+
+  public String getName() {
+    return region.id();
```

Comment:
Question: why not `getId`?

Reply:
or should this be returning the `regionName` variable?

Reply:
Removed as this function is not used.